backend/workflow_manager/endpoint/destination.py

```diff
@@ -140,37 +163,34 @@ def handle_output(
             if connection_type == WorkflowEndpoint.ConnectionType.API:
                 self._handle_api_result(file_name=file_name, error=error, result=result)
             return
+        file_history = FileHistoryHelper.get_file_history(
+            workflow=workflow, cache_key=file_hash.file_hash
+        )
         if connection_type == WorkflowEndpoint.ConnectionType.FILESYSTEM:
             self.copy_output_to_output_directory()
         elif connection_type == WorkflowEndpoint.ConnectionType.DATABASE:
-            self.insert_into_db(
-                file_history=file_history, input_file_path=input_file_path
-            )
+            if (
+                file_hash.file_destination
+                == WorkflowEndpoint.ConnectionType.MANUALREVIEW
+            ):
+                self._push_data_to_queue(file_name, workflow, input_file_path)
+            else:
+                self.insert_into_db(input_file_path=input_file_path)
         elif connection_type == WorkflowEndpoint.ConnectionType.API:
             result = self.get_result(file_history)
             meta_data = self.get_metadata(file_history)
             self._handle_api_result(
                 file_name=file_name, error=error, result=result, meta_data=meta_data
             )
         elif connection_type == WorkflowEndpoint.ConnectionType.MANUALREVIEW:
-            if file_history:
-                logger.info(
-                    f"Already processed result, skip adding to MRQ. "
-                    f"WorkflowID{workflow.id}"
-                )
-                return
-            result = self.get_result(file_history)
-            meta_data = self.get_metadata(file_history)
-            self._push_to_queue(
-                file_name=file_name,
-                workflow=workflow,
-                result=result,
-                input_file_path=input_file_path,
-                meta_data=meta_data,
+            self._push_data_to_queue(file_name, workflow, input_file_path)
+        if self.execution_service:
+            self.execution_service.publish_log(
+                message=f"File {file_name} processed successfully"
```

Comment:
```suggestion
                message=f"File {file_name} pushed to manual review queue."
```

Reply:
@chandrasekharan-zipstack this is not part of MRQ. This was already there.